python/cuml/svm/svm.pyx

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
Need to provide a comprehensive set of pytests to cover as many paths through SVM as possible:
- test each kernel type with a relevant dataset
- test general linear classification on a series of datasets
- test different input types (numba, numpy, dataframe, etc..) and output types
- test different feature matrix shapes (tall skinny, short fat, tall fat, etc...)
- test different input precisions
- test different param configurations, including isolating tolerances that might cause an underflow
- test pickling
- test running end to end fit() and predict() in a loop w/ many iterations (100+) to test for possible memory leaks (from both cython and c++)

The  GTests should also cover many of the above, with the exception of pickling and input/output types.

Reply:
Task list for tests:
- [x] test each kernel type with a relevant dataset
- [ ] test general linear classification on a series of datasets
- [x] test different input types (numba, numpy, dataframe, etc..)
- output types are not configurable, currently prediction returns cuDF series, I would wait until Issue #637 is solved with the implementation of different output types. 
- [ ] test different feature matrix shapes (tall skinny, short fat, tall fat, etc...)
- [x] test different input precisions
- [x] test different param configurations, 
- [ ]  isolating tolerances that might cause an underflow
- [x] test pickling
- [ ] test running end to end fit() and predict() in a loop w/ many iterations (100+) to test for possible memory leaks

Reply:
I still have to tune the parameters and sensitivity for the skl_cmp_dataset test. What is the walltime budget for quality and stress tests? 

The memory leak test is not conclusive yet, I will investigate memory usage and then improve that test.